web/client/configs/localConfig.json

```diff
@@ -412,12 +412,6 @@
     ],
     "desktop": [
       "Details",
-      {
-        "name": "BrandNavbar",
-        "cfg": {
-          "containerPosition": "header"
-        }
-      },
```

Comment:
Why are we removing BrandNavbar from configuration?
The default application should keep the plugin while in the datadir we should just add the override for the specific cfg to override the logo

Reply:
I agree with @allyoucanmap.  think we need you @offtherailz for a quick exchange regarding possible solutions on the MS backend service that manage the configuration returned to the client.

Reply:
I had a call with @offtherailz, a possible solution is to implement support of `jsonpath` backend side to allow changes on items inside arrays, this kind of implementation needs an investigation to understand the feasibility (there was already an issue #9278).